<inh f='include/c++/4.9/exception' l='60' c='std::exception'/>
<def f='include/c++/4.9/stdexcept' l='55'/>
<ovr f='include/c++/4.9/stdexcept' l='74' c='std::domain_error'/>
<ovr f='include/c++/4.9/stdexcept' l='82' c='std::invalid_argument'/>
<ovr f='include/c++/4.9/stdexcept' l='91' c='std::length_error'/>
<ovr f='include/c++/4.9/stdexcept' l='100' c='std::out_of_range'/>
<size>16</size>
<doc f='include/c++/4.9/stdexcept' l='50'>/** Logic errors represent problems in the internal logic of a program;
   *  in theory, these are preventable, and even detectable before the
   *  program runs (e.g., violations of class invariants).
   *  @brief One of two subclasses of exception.
   */</doc>
<ovr f='include/boost/asio/io_service.hpp' l='721' c='boost::asio::service_already_exists'/>
<ovr f='include/boost/asio/io_service.hpp' l='730' c='boost::asio::invalid_service_owner'/>
<ovr f='include/boost/program_options/errors.hpp' l='32' c='boost::program_options::error'/>
